packages/dbml-core/types/parse/error.d.ts

```diff
@@ -0,0 +1,29 @@
+export declare type WarningLevel =
+  | 'error'
+  | 'warning'
+  | 'info';
```

Comment:
It's currently not necessary, but monaco does support yellow markers for example instead of red markers, like you see in linting tools. This indicates the nature of the error. if the `type` field is `undefined` simply assume it an `error`.